docs/content/en/docs-dev/contribution-guidelines/development.md

```diff
@@ -33,7 +33,7 @@ For more information, you might want to read the [installation notes of `bazelis
 - `make buildifier`: formats bazel BUILD and .bzl files with a standard convention.
 - `make clean`: cleans all bazel cache.
 - `make expose-generated-go`: exposes generated Go files (`.pb.go`, `.mock.go`...) to editors and IDEs.
-- `make site`: runs PipeCD site (https://pipecd.dev) locally (requires [hugo](https://github.com/gohugoio/hugo) with `_extended` version `0.88.1` or later to be installed).
```

Comment:
we're not going to release a new version soon, so I think it's better to update docs under `/docs` and `/docs-v0.25.x` as well.